package.json

```
@@ -11,14 +11,13 @@
     "test": "jest"
   },
   "dependencies": {
-<<<<<<< feat/#21/signup-ui
     "@hookform/resolvers": "^5.2.2",
-=======
     "date-fns": "^4.1.0",
```

Biome (2.4.15)

[error] The key date-fns was already declared.

(lint/suspicious/noDuplicateObjectKeys)